content/docs/rfcs/52/README.md

```diff
@@ -18,13 +18,6 @@ This document lists static shard flag index assignments (see [51/WAKU2-RELAY-SHA
 Similar to the [IANA port allocation](https://www.iana.org/assignments/service-names-port-numbers/service-names-port-numbers.xhtml),
 this document lists static shard index assignments (see [51/WAKU2-RELAY-SHARDING](/spec/51/).
 
-The index ranges are as follows:
```

Comment:
removing this since its already defined in RFC51. always only one source of truth.